lib/ace/lib/event.js

```diff
@@ -37,25 +37,24 @@ var useragent = require("./useragent");
 var pressedKeys = null;
 var ts = 0;
 
+var activeListenerToken = false;
```

Comment:
Can we do this lazily instead of executing the code in the global scope? I prefer to have "require" have as little side effects as possible.